controller/RestDelivery.php

```diff
@@ -254,6 +254,62 @@ public function deleteDeferred()
         }
     }
 
+    /**
+     * List all deliveries or paginated range
+     */
+    public function get()
+    {
+        try {
+            if ($this->getRequestMethod() !== \Request::HTTP_GET) {
+                throw new \common_exception_NotImplemented('Only get method is accepted to getting deliveries');
+            }
+
+            $limit = 0;
+            if ($this->hasRequestParameter('limit')) {
+                $limit = $this->getRequestParameter('limit');
+                if (!is_numeric($limit) || $limit < 0) {
```

Comment:
To make sure it's not a float
if (!is_numeric($offset) || intval($offset) != $offset || $offset < 0) {